### DISCUSSION TITLE
Auxilary part cooling fan

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1280] [General]
Hey guys, is it possible to turn an auxiliary part cooling fan on (or off) using your Professional Firmware?  I use the firmware today with the BTT SKR Mini 3 v3 and I love it!    Maybe in the slicer?  I do not know yet.   No worries if this is not possible, but thought I'd ask before installing auxiliary part cooling fan. I do have options of installing better cooling on the extruder mechanism.    TYVM

**Comment by mriscoc** [comment]
Hi, could be possible but you need to configure and recompile Marlin, I suggest you to try with official Marlin first.
